core/lib/dal/src/tokens_dal.rs

```diff
@@ -78,13 +78,12 @@ impl TokensDal<'_, '_> {
         }
     }
 
-    pub async fn get_well_known_token_addresses(&mut self) -> Vec<(Address, Address)> {
+    pub(crate) async fn get_well_known_token_addresses(&mut self) -> Vec<(Address, Address)> {
```

Comment:
There is also `get_well_known_tokens` in `tokens_web3_dal` where we should remove filter by `well_known`

Reply:
Done.